README.md

```diff
@@ -288,8 +301,17 @@ Read our [privacy statement](https://developers.redhat.com/article/tool-data-col
 This plugin respects the settings of the `Telemetry by Red Hat` plugin, which you can learn more
 about [here](https://plugins.jetbrains.com/plugin/16209-telemetry-by-red-hat).
 
+
 ## Support, feedback & questions
 
+The Red Hat Dependency Analytics plugin for IntellJ IDEA in current version, supports the following Intelli-J IDE Versions:
```

Comment:
Typo, missing an "i".

`... IntellJ IDEA ...` --> `... IntelliJ IDEA ...`

I don't think there is a dash (-) in "IntelliJ", at least I don't see it that way on their website.  I would remove the dash.  Also, add an "A" to "IDE", and no need to capitalize the word "Versions".

`... Intelli-J IDE Versions ...` --> `... IntelliJ IDEA versions ...`